Makefile

```diff
@@ -4,3 +4,11 @@ test:
 	go test -v -count=1 -race -cover ./...
 
 .PHONY: test
+
+audit:
+	go list -json -m all | nancy sleuth --exclude-vulnerability-file ./.nancy-ignore
```

Comment:
No need to specify the exclude file, `.nancy-ignore` is already the default anyway so this flag does nothing.
```suggestion
	go list -json -m all | nancy sleuth
```